sdk/storage/azure-storage-queue/azure/storage/queue/_message_encoding.py

```diff
@@ -7,36 +7,45 @@
 
 import sys
 from base64 import b64encode, b64decode
+from typing import Any, Dict, TYPE_CHECKING
 
 from azure.core.exceptions import DecodeError
 
 from ._encryption import decrypt_queue_message, encrypt_queue_message, _ENCRYPTION_PROTOCOL_V1
 
+if TYPE_CHECKING:
+    from azure.core.pipeline import PipelineResponse
+
 
 class MessageEncodePolicy(object):
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.require_encryption = False
         self.encryption_version = None
         self.key_encryption_key = None
         self.resolver = None
 
-    def __call__(self, content):
+    def __call__(self, content: object) -> str:
         if content:
             content = self.encode(content)
             if self.key_encryption_key is not None:
                 content = encrypt_queue_message(content, self.key_encryption_key, self.encryption_version)
         return content
 
-    def configure(self, require_encryption, key_encryption_key, resolver, encryption_version=_ENCRYPTION_PROTOCOL_V1):
+    def configure(
+        self, require_encryption: bool,
+        key_encryption_key: str,
+        resolver: callable,
+        encryption_version: str = _ENCRYPTION_PROTOCOL_V1
+    ) -> None:
         self.require_encryption = require_encryption
         self.encryption_version = encryption_version
         self.key_encryption_key = key_encryption_key
         self.resolver = resolver
         if self.require_encryption and not self.key_encryption_key:
             raise ValueError("Encryption required but no key was provided.")
 
-    def encode(self, content):
+    def encode(self, content: object) -> None:
```

Comment:
Return type should be `str`

Reply:
Let's make content `Any` for top-level, and then for our implementations lets make it match what it actually is in its implementation